tools/cldr-code/src/test/java/org/unicode/cldr/unittest/LikelySubtagsTest.java

```diff
@@ -349,6 +356,30 @@ public void TestMissingInfoForLanguage() {
                 }
                 errln("Missing English translation for: " + language + " which is at " + covLevel);
             }
+            // now, check general coverage level. Use "und" so it is neutral.
```

Comment:
This is the check that should only be done if the **last-release's** effective coverage level is > Modern.